src/main/java/crawlercommons/sitemaps/AbstractSiteMapSAXHandler.java

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2016 Crawler-Commons
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package crawlercommons.sitemaps;
+
+import java.net.URL;
+import java.util.LinkedList;
+
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.SAXParseException;
+import org.xml.sax.helpers.DefaultHandler;
+
+/**
+ * Provides a base SAX handler for parsing of XML documents representing
+ * sub-classes of AbstractSiteMap.
+ */
+public class AbstractSiteMapSAXHandler extends DefaultHandler {
```

Comment:
I agree. Would DefaultSiteMapSAXHandler be a better name? or rename SiteMapSAXHandler into  something else and use SiteMapSAXHandler for this one?